views/pod.html

```diff
@@ -15,7 +15,7 @@
     <dt>Phase</dt>
     <dd>{{resource.status.phase}}</dd>
     <dt>Node</dt>
-    <dd>{{resource.status.host}}</dd>
+    <dd>{{resource.spec.host}}</dd>
```

Comment:
Could change it to be:

hostname (ip)   - when both exists and are not the same
hostname - when both exist and are the same (usually this is when hostname is actually the ip)
ip - when hostname doesn't exist and ip is set
unknown - when neither one is set